docs/source/Analysis.rst

```diff
@@ -25,11 +25,30 @@ Configuring Analysis
 --------------------
 .. _configure analysis:
 
-.. automodule:: bom_analysis.base.BaseConfigMethods
```

Comment:
Have you moved these API docs elsewhere? I don't see them. While auto-API docs are not sufficient, you'd typically still want to make them available as part of your documentation.

Reply:
I have not removed them, I replaced the automodule with the text it pointed at in the .rst and then linked the .rst in the API docs